apps/wpcom-block-editor/src/utils.js

```diff
@@ -59,14 +61,14 @@ export const isEditorReady = async () =>
 export const isEditorReadyWithBlocks = async () =>
 	new Promise( ( resolve ) => {
 		const unsubscribe = subscribe( () => {
-			const isCleanNewPost = select( 'core/editor' ).isCleanNewPost();
+			const isCleanNewPost = select( editorStore ).isCleanNewPost();
 
 			if ( isCleanNewPost ) {
 				unsubscribe();
 				resolve( false );
 			}
 
-			const blocks = select( 'core/editor' ).getBlocks();
+			const blocks = select( blockEditorStore ).getBlocks();
```

Comment:
Did you mean `editorStore` here? 

```suggestion
			const blocks = select( editorStore ).getBlocks();
```